assets/xml/objects/object_mk.xml

```diff
@@ -1,45 +1,46 @@
-        <Texture Name="object_mk_Tex_006020" OutName="tex_006020" Format="ci8" Width="16" Height="16" Offset="0x6020" />
-        <Texture Name="object_mk_Tex_006120" OutName="tex_006120" Format="ci8" Width="8" Height="8" Offset="0x6120" />
-        <Texture Name="object_mk_Tex_006160" OutName="tex_006160" Format="ci8" Width="8" Height="8" Offset="0x6160" />
-        <Texture Name="object_mk_Tex_0061A0" OutName="tex_0061A0" Format="ci8" Width="8" Height="16" Offset="0x61A0" />
-        <Texture Name="object_mk_Tex_006220" OutName="tex_006220" Format="rgba16" Width="32" Height="32" Offset="0x6220" />
-        <Texture Name="object_mk_Tex_006A20" OutName="tex_006A20" Format="ci8" Width="8" Height="8" Offset="0x6A20" />
-        <Texture Name="object_mk_Tex_006A60" OutName="tex_006A60" Format="ci8" Width="8" Height="8" Offset="0x6A60" />
-        <Texture Name="object_mk_Tex_006AA0" OutName="tex_006AA0" Format="ci8" Width="8" Height="8" Offset="0x6AA0" />
-        <Texture Name="object_mk_Tex_006AE0" OutName="tex_006AE0" Format="ci8" Width="16" Height="16" Offset="0x6AE0" />
-        <Limb Name="object_mk_Standardlimb_006BE0" Type="Standard" EnumName="OBJECT_MK_LIMB_01" Offset="0x6BE0" />
-        <Limb Name="object_mk_Standardlimb_006BEC" Type="Standard" EnumName="OBJECT_MK_LIMB_02" Offset="0x6BEC" />
-        <Limb Name="object_mk_Standardlimb_006BF8" Type="Standard" EnumName="OBJECT_MK_LIMB_03" Offset="0x6BF8" />
-        <Limb Name="object_mk_Standardlimb_006C04" Type="Standard" EnumName="OBJECT_MK_LIMB_04" Offset="0x6C04" />
-        <Limb Name="object_mk_Standardlimb_006C10" Type="Standard" EnumName="OBJECT_MK_LIMB_05" Offset="0x6C10" />
-        <Limb Name="object_mk_Standardlimb_006C1C" Type="Standard" EnumName="OBJECT_MK_LIMB_06" Offset="0x6C1C" />
-        <Limb Name="object_mk_Standardlimb_006C28" Type="Standard" EnumName="OBJECT_MK_LIMB_07" Offset="0x6C28" />
-        <Limb Name="object_mk_Standardlimb_006C34" Type="Standard" EnumName="OBJECT_MK_LIMB_08" Offset="0x6C34" />
-        <Limb Name="object_mk_Standardlimb_006C40" Type="Standard" EnumName="OBJECT_MK_LIMB_09" Offset="0x6C40" />
-        <Limb Name="object_mk_Standardlimb_006C4C" Type="Standard" EnumName="OBJECT_MK_LIMB_0A" Offset="0x6C4C" />
-        <Limb Name="object_mk_Standardlimb_006C58" Type="Standard" EnumName="OBJECT_MK_LIMB_0B" Offset="0x6C58" />
-        <Limb Name="object_mk_Standardlimb_006C64" Type="Standard" EnumName="OBJECT_MK_LIMB_0C" Offset="0x6C64" />
-        <Skeleton Name="object_mk_Skel_006CA0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_MK_LIMB_NONE" LimbMax="OBJECT_MK_LIMB_MAX" EnumName="ObjectMkLimb" Offset="0x6CA0" />
+        <Animation Name="gMarineResearcherHeadWaggleAnim" Offset="0x438" />
+        <Animation Name="gMarineResearcherYellingAnim" Offset="0x7D8" />
+        <Animation Name="gMarineResearcherShakingInFearAnim" Offset="0x10F4" />
+        <Animation Name="gMarineResearcherStrokingChinAnim" Offset="0x1964" />
```

Comment:
```suggestion
        <Animation Name="gMarineResearcherYellAnim" Offset="0x7D8" />
        <Animation Name="gMarineResearcherShakeInFearAnim" Offset="0x10F4" />
        <Animation Name="gMarineResearcherStrokeChinAnim" Offset="0x1964" />
```
We tend to avoid `ing` in anim names like this

Reply:
Gotcha. Changed to your suggestion